packages/react-storage/src/components/StorageBrowser/components/DescriptionList.tsx

```diff
@@ -11,7 +11,7 @@ import { CLASS_BASE } from '../views/constants';
 
 export interface DescriptionItemProps {
   term?: string;
-  details?: string;
+  details?: string | JSX.Element;
```

Comment:
Why this change?

Reply:
🍕

Reply:
to support passing jsx as the detail, but we can revisit whether putting the the breadcrumbs in a description list is correct from an accessibility perspective.

Reply:
Caring about accessibility project the best man asking women

Reply:
To eat 🍕 everynight

Reply:
Let's follow up on this. Don't think it makes sense with the approach for a11y or composability